feat: add ai service kit directory and kit descriptions

izuru0 opened the pull request:
## Description

- this PR introduces a new kit, the ~~LLM Support KIT~~AI Service KIT (changed to more generic term), which defines how AI systems can collaborate in dataspace ecosystem
- this commit adds adoption view, software development view and changelog pages in newly created docs-kits/kits/llm-support-kit subdirectory
- issue [1548](https://github.com/eclipse-tractusx/sig-release/issues/1506) of sig-release repository is the corresponding issue

<!-- Please tag the related issue `Fixes or Updates #issue_number`, if applicable. -->

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

izuru0 commented on docs-kits/kits/llm-support-kit/adoption-view.md:
```diff
@@ -0,0 +1,65 @@
+
+### Risk Avoidance
+
+Business units considering AI system adoption must address AI-specific considerations, including compliance with AI-related laws, regulations, and guidelines such as the AI Act.
```
@hschwabe 
fixed this in commit 13393ac

matbmoser commented:
I am taking over the review for this KIT. Please specify a release ticket so I can start and document the review.

hokonohara commented:
Please change the picture, the "AI System" should be on the right side (the provider).

matbmoser commented on docs-kits/kits/ai-support-kit/resources/topology-small.png:
For each image we need to add a copyright, look in TRG 7.07: https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-07

You can create one per folder.

Here they are referenced: https://eclipse-tractusx.github.io/docs/release/trg-10/trg-10-02#mandatory-trgs

Images all must be stored in the resources folder, recommended under /resources/img

matbmoser commented on docs-kits/kits/ai-support-kit/concept-ai-small.png:
this image shall be in the stored in the resources folder (or under /resources/img)

izuru0 replied:
@matbmoser 

all images are now in `resources/img/`

matbmoser commented on docs-kits/kits/ai-support-kit/glossary.md:
glossary can be included under the adoption view.

Here is how the structure shall look like:

docs-kits/kits/
  <kit-name>/
    ├── changelog.md                       # Version history following semantic versioning
    ├── adoption-view/
      │   ├── _category_.json
      │   └── adoption-view.md               # Business value, use cases, policies, standards
      │   └── glossary.md                         # Your glossary
      ├── development-view/
      │   ├── _category_.json
      │   └── development-view.md            # Architecture, API specs, semantic models, tutorials
      └── resources/
          └── img/                           # KIT-specific images and diagrams (with their licenses) 
                      └── REUSE.toml (look at the [TRG 7.07](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-07)
          └── ...

matbmoser reviewed (changes requested):
@izuru0 and @hokonohara please update this things (see my comments below)

One more thing is missing is to add it to the master data. (I would include this in Industry Core Foundation)

Also we are missing a logo. I can provide you one.

stephanbcbauer commented:
Please be aware, the changes need to be merged tomorrow until 1 pm to be part of release 25.12. Thx. Please also don't forget the adaptation of the KIT master data file

izuru0 commented:
@matbmoser 

in my latest commit 5e5f40e I added a line stating that development view is still a draft

I talked with experts in Catena-X I Trusted AI Expert Group and they agreed that the content in this version is ok and can be merged

izuru0 commented:
@matbmoser 

I believe I corrected items raised in this pull request comments section.
If there are anything I missed, let me know.

matbmoser reviewed (changes requested):
Review on 11.12.2025:

- Compilation issues in the website (markdown linting and build)
- No KIT icon yet available (@Grand-Thibault would love if you could support here): follow guidelines from here: [TRG 10.02 KIT Content - LOGO](https://eclipse-tractusx.github.io/docs/release/trg-10/trg-10-02#kit-logo)
- Missing "Contributors to the Eclipse Foundation" in the copyright as mentioned here: https://eclipse-tractusx.github.io/documentation/kit-framework#copyright-notice
   - Just applies for the views, not for the images (the images can contain just you)
   - If you have been working with more companies please include their copyright too.
   - If you dont want to mention them explicitly, include Contributors to the Eclipse Foundation

Please join the KIT office hours so I can support you, next one will be happening next week Thursday (last one in this year).

https://eclipse-tractusx.github.io/community/open-meetings#Eclipse%20Tractus-X%20KITs%20Community%20Office%20Hour

izuru0 commented:
@matbmoser 
- fixed lint problems
- added copyright lines. I will ask Trust AI experts again about copyright notice
- thank you for the logo but I cannot see it (I tried from different PC and iOS app) at the moment. I will try later again
  <img width="1627" height="1122" alt="image" src="https://github.com/user-attachments/assets/e0c2d2a4-f056-4087-8faf-3daec0775b8e" />
- thank you for fixing the title of this pull request (LLM --> AI)
- about the name: I prefer a name like 'AI Service KIT' over 'Federated', because we want to support cases @hokonohara mentioned as well

matbmoser commented:
<img width="1624" height="428" alt="image" src="https://github.com/user-attachments/assets/d1dfd958-3e10-47d3-8613-b0f4dcede411" />
still are broken links, I will fix it for you.

matbmoser commented on docs-kits/kits/ai-service-kit/adoption-view/adoption-view.md:
```diff
@@ -0,0 +1,75 @@
+
+## Use Case
+
+TBD
```
This I will remove.

matbmoser commented:
Since there was no update her I will merge it, it is fine we can still decide if we change something and if we take into the release.
